src/api/organisationUnits-dataEngine.js

```diff
@@ -0,0 +1,115 @@
+import { onError } from './index'
```

Comment:
Is the goal to remove the old `api/organisationUnits` as soon as Dashboards is converted and OrgUnitDimension is redone with these functions?

Reply:
Actually this one with the `dataEngine` postfix will be renamed to `organisationUnits` and replace the old one.
For now we need both.